src/rewards/RewardsStorage.sol

```diff
@@ -24,6 +24,7 @@ abstract contract RewardsStorage is BrevisProofApp, AccessControl {
     );
     event ProofProcessed(bytes32 indexed proofId, uint32 indexed epoch, uint32 indexed batchIndex);
     event RewardsAdjusted(uint64 indexed adjustmentId, address indexed user, int256[] adjustment);
+    event RewardsSet(address indexed user, uint256[] cumulativeRewards);
```

Comment:
Emit token address in all reward-related events?

Reply:
Why? It's fixed on chain, and repetitive for all users. Most campaign only has one token